src/SprykerSdk/Spryk/Model/Spryk/Builder/ResourceRoute/ResourceRouteSpryk.php

```diff
@@ -0,0 +1,240 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Builder\ResourceRoute;
+
+use Roave\BetterReflection\BetterReflection;
+use SprykerSdk\Spryk\Exception\EmptyFileException;
+use SprykerSdk\Spryk\Exception\ReflectionException;
+use SprykerSdk\Spryk\Model\Spryk\Builder\SprykBuilderInterface;
+use SprykerSdk\Spryk\Model\Spryk\Builder\Template\Renderer\TemplateRendererInterface;
+use SprykerSdk\Spryk\Model\Spryk\Definition\SprykDefinitionInterface;
+use SprykerSdk\Spryk\Style\SprykStyleInterface;
+
+class ResourceRouteSpryk implements SprykBuilderInterface
+{
+    public const ARGUMENT_TARGET = 'target';
+    public const ARGUMENT_TEMPLATE = 'template';
+    public const ARGUMENT_METHOD_NAME = 'method';
+    protected const ARGUMENT_RESOURCE_ROUTE_METHOD = 'resourceRouteMethod';
+
+    /**
+     * @var \SprykerSdk\Spryk\Model\Spryk\Builder\Template\Renderer\TemplateRendererInterface
+     */
+    protected $renderer;
+
+    /**
+     * @param \SprykerSdk\Spryk\Model\Spryk\Builder\Template\Renderer\TemplateRendererInterface $renderer
+     */
+    public function __construct(TemplateRendererInterface $renderer)
+    {
+        $this->renderer = $renderer;
+    }
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return 'resourceRoute';
+    }
+
+    /**
+     * @param \SprykerSdk\Spryk\Model\Spryk\Definition\SprykDefinitionInterface $sprykDefinition
+     *
+     * @return bool
+     */
+    public function shouldBuild(SprykDefinitionInterface $sprykDefinition): bool
+    {
+        return (!$this->isRouteDeclared($sprykDefinition));
+    }
+
+    /**
+     * @param \SprykerSdk\Spryk\Model\Spryk\Definition\SprykDefinitionInterface $sprykDefinition
+     * @param \SprykerSdk\Spryk\Style\SprykStyleInterface $style
+     *
+     * @return void
+     */
+    public function build(SprykDefinitionInterface $sprykDefinition, SprykStyleInterface $style): void
+    {
+        $targetFileContent = $this->getTargetFileContent($sprykDefinition);
+
+        $templateName = $this->getTemplateName($sprykDefinition);
+
+        $methodContent = $this->renderer->render(
+            $templateName,
+            $sprykDefinition->getArgumentCollection()->getArguments()
+        );
+
+        $search = 'return $resourceRouteCollection;';
+        $positionOfReturnStatement = strrpos($targetFileContent, $search);
+
+        if ($positionOfReturnStatement !== false) {
```

Comment:
Please use `if ($positionOfReturnStatement)` instead.

Reply:
`strrpos()` will 

> Returns FALSE if the needle was not found.

This check is checking if the `$search` is not found.